packages/fxa-auth-server/lib/server.js

```diff
@@ -366,17 +356,6 @@ async function create(
   );
   server.auth.strategy('subscriptionsSecret', 'subscriptionsSecret');
 
-  server.auth.scheme(
-    'fxa-oauthServerJWT',
```

Comment:
This was used by an oauth->auth request for the profile, which got replaced at some point last year with a direct db call